.changelog/2529.feature.md

```diff
@@ -0,0 +1,3 @@
+`RuntimeID` is not hardcoded anymore in the enclave, but is passed when
```

Comment:
This is a breaking change as it breaks the runtime protocol. Change the type and mention in a paragraph that it breaks the runtime protocol.

This also requires a runtime protocol version bump (in both Go and Rust).